LEAF_Request_Portal/open.php

```diff
@@ -5,7 +5,7 @@
 
 error_reporting(E_ERROR);
 
-include '../libs/loaders/Leaf_autoloader.php';
+require_once '../libs/loaders/Leaf_autoloader.php';
 
 $report = isset($_GET['report']) ? Leaf\XSSHelpers::xscrub($_GET['report']) : '';
 if($report != '') {
```

Comment:
I am not sure when this was actually introduced, but it looks like at some point the shortener class got shortened...

Reply:
Found and fixed in step 4, merged into this step also.